src/components/BulletCursorOverlay.tsx

```diff
@@ -0,0 +1,135 @@
+import { useRef } from 'react'
+import { useSelector } from 'react-redux'
+import { css } from '../../styled-system/css'
+import Path from '../@types/Path'
+import SimplePath from '../@types/SimplePath'
+import useBulletPosition from '../hooks/useBulletPosition'
+import attributeEquals from '../selectors/attributeEquals'
+import isContextViewActive from '../selectors/isContextViewActive'
+import rootedParentOf from '../selectors/rootedParentOf'
+import head from '../util/head'
+import parentOf from '../util/parentOf'
+
+/** A larger circle that surrounds the bullet of the cursor thought. */
+const BulletCursorOverlay = ({
+  isCursorActive,
+  x = 0,
+  y = 0,
+  simplePath,
+  path,
+}: {
+  isCursorActive?: boolean
+  x?: number
+  y?: number
+  simplePath?: SimplePath
+  path?: Path
```

Comment:
Similarly, `simplePath` is derivable from `path` (via `simplifyPath(state, path)`), so you should remove it from the hook arguments and calculate it internally in a `useSelector`.